sbin/pfdhcplistener

```diff
@@ -339,10 +341,22 @@ sub process_pkt {
                     $logger->debug("[$dhcp_mac] Ignoring packet due to rate-limiting ($rate_limit_key)");
                 }
             }
-            else {
-                my $apiclient = pf::api::queue->new;
+
+            # IPv6 processing
+            elsif ( $l2->{type} eq ETH_TYPE_IPv6 ) {
+                $pf::StatsD::statsd->increment("pfdhcplistener_$statsd_interface\::process_pkt_ipv6.count" );
+
+                # TODO: Rate-limiting for IPv6 just as for IPv4
+
+                my $apiclient = pf::api::queue->new(queue => 'pfdhcplistener');
                 $apiclient->notify('process_dhcpv6', MIME::Base64::encode($l4->{data}));
             }
+
+            # Non IPv4 nor IPv6 packet
+            else {
+                $logger->warn("Skipping non IPv4 nor IPv6 DHCP packet from MAC address '$args{'src_mac'}' (IP: '$args{'src_ip'}') for IP '$args{'dest_ip'}'");
```

Comment:
Not sure if thats going to pollute the logs with special STP traffic or these type of things